### ISSUE TITLE
Is macos big sur working?

### ISSUE THREAD

**Issue by bluebenzene** [allansrc/Catalina-macOS-Aspire-5#6]
please confirm that ,then after i'll try to install 👍

**Comment by allansrc** [comment]
Yep, on iMacPro1,1 it worked smoothly, i do not test with MacPro7,1 SMBios yet.

Remember to use MacProMemoryNotificationDisabler.kext on MacPro7,1 after installs it

**Comment by allansrc** [comment]
Big sur is not stable, some issues and KP still on.
But i am currently working on change this repo for OpenCore this week or the next i will update it

**Comment by bluebenzene** [comment]
Thanks. Big Sur is Beautiful, but  I stick to CATALINA until it became stable.
